src/core/processing/qgsprocessingfeedback.cpp

```diff
@@ -67,7 +67,7 @@ void QgsProcessingFeedback::pushVersionInfo()
 
 #if PROJ_VERSION_MAJOR > 4
   PJ_INFO info = proj_info();
-  pushDebugInfo( tr( "PROJ version: %1.%2.%3" ).arg( PROJ_VERSION_MAJOR ).arg( PROJ_VERSION_MINOR ).arg( PROJ_VERSION_PATCH );
+  pushDebugInfo( tr( "PROJ version: %1.%2.%3" ).arg( PROJ_VERSION_MAJOR ).arg( PROJ_VERSION_MINOR ).arg( PROJ_VERSION_PATCH ) );
```

Comment:
??? I do not have this error in my updated master

Reply:
is because has already fixed by https://github.com/qgis/QGIS/commit/2a6629ef56946e3ee54f81571350c7a6b94e6339#commitcomment-32999050